lib/qs/test_runner.rb

```diff
@@ -40,4 +36,41 @@ def normalize_params(params)
 
   end
 
+  class JobTestRunner < TestRunner
+
+    def initialize(handler_class, args = nil)
+      if !handler_class.include?(Qs::JobHandler)
+        raise InvalidJobHandlerError, "#{handler_class.inspect} is not a"\
+                                      " Qs::JobHandler"
+      end
+
+      super(handler_class, args)
+    end
+
+  end
+
+  class EventTestRunner < TestRunner
+
+    def initialize(handler_class, args = nil)
+      if !handler_class.include?(Qs::EventHandler)
+        raise InvalidEventHandlerError, "#{handler_class.inspect} is not a"\
+                                      " Qs::EventHandler"
+      end
+
+      args         = (args || {}).dup
+      channel      = args.delete(:event_channel) || Factory.string
+      name         = args.delete(:event_name)    || Factory.string
```

Comment:
@jcredding don't think we can rely on `Factory` here.  We discussed just using 'a-channel' and 'a-name'?